common/lib/common-utils/.eslintrc.js

```diff
@@ -5,17 +5,19 @@
 
 module.exports = {
     "extends": [
-        "@fluidframework/eslint-config-fluid"
+        "@fluidframework/eslint-config-fluid/eslint7"
     ],
     "parserOptions": {
         "project": [ "./tsconfig.json", "./test/tsconfig.json" ]
     },
     "rules": {
         "@typescript-eslint/no-non-null-assertion": "off",
+        "@typescript-eslint/no-unsafe-return": "off",
+        "@typescript-eslint/no-unused-vars": "off",
```

Comment:
did these two warnings produce a lot of errors in common-utils/is there an issue to clean them up and remove these warning suppressions?

Reply:
@heliocliu Thanks - I'm not sure how I wound up suppressing these both at the project level. I fixed that. There aren't too many offenses. I tried to add typings for sha.js and change the import in hashFileNode.ts to `import { sha1 } from "sha.js";` but that brings in node types which breaks the build.

If we had proper types for sha.js that would remove a couple of the unsafe returns.